scripts/hourly_wmb.py

```diff
@@ -77,5 +80,12 @@
 
     # Pipeline
     for file in download(args):
-        for row in normalize(file):
+
+        if args.cache_file:
+            to_cache, file_stream = tee(file)
+            with open(args.cache_file, 'w') as f:
+                for line in to_cache:
+                    f.write(line)
+
+        for row in normalize(file_stream):
```

Comment:
I *think* that this will fail, if `args.cache_file` is not defined. Because then `file_stream` will not be defined...

Reply:
uh oh, yes it absolutely will, I'll change it such that file_stream will just be named file.